### ISSUE TITLE
Fix needed [Code] setup_channels.sh script assumes jq is installed

### ISSUE THREAD

**Issue by vnprc** [lnbook/lnbook#318]
**Describe the error**
the setup_channels.sh script appears to work for me but I get errors indicating that it relies upon jq being installed
```
$ bash setup-channels.sh 
Getting node IDs
setup-channels.sh: line 4: jq: command not found
setup-channels.sh: line 5: jq: command not found
setup-channels.sh: line 6: jq: command not found
setup-channels.sh: line 7: jq: command not found
Alice:
Bob:
Wei:
Gloria:
Setting up channels...
Alice to Bob
[lncli] rpc error: code = Unknown desc = pubkey string is empty
[lncli] unable to decode node public key: encoding/hex: odd length hex string
Bob to Wei
{
   "code": -32602,
   "message": "id \"@ not valid"
}
{
   "code": -32602,
   "message": "'id' should be a node id, not '1000000'"
}
Wei to Gloria
The form field 'uri' was malformed:
Invalid node id
The form field 'nodeId' was malformed:
requirement failed
Get 10k sats invoice from Gloria
setup-channels.sh: line 43: jq: command not found
Gloria invoice
Wait for channel establishment - 60 seconds for 6 blocks
Alice pays Gloria 10k sats, routed around the network
[lncli] pay_req argument missing
```

**To Fix**
Recommend you update the script to use bash -c args or otherwise instruct the user to install jq locally.

**Comment by vnprc** [comment]
https://github.com/lnbook/lnbook/pull/319

**Comment by aantonop** [comment]
Fixed by #319
